player_app/urls.py

```diff
@@ -0,0 +1,11 @@
+from django.urls import path, include
+from .views import PlayerDetails, PlayerTraitsView, GameProgress
+
+urlpatterns = [
+#     path('profile/', PlayerDetails.as_view()),
+#     path('traits/', PlayerTraits.as_view()),
+#     path('progress/', GameProgress.as_view()),
+    path('profile/<int:player_id>/', PlayerDetails.as_view()),
```

Comment:
Why do you need the player id? It may be better to grab the player through `request.player` and utilize token authentication to authenticate the player upon each request. You are exposing a users primary key in your database every time you make a request every time you do it this way